generator.yaml

```diff
@@ -17,12 +18,131 @@ ignore:
   shape_names: null
 model_name: route53
 operations:
+  ChangeResourceRecordSets:
+    operation_type:
+    - Create
+    - Delete
+    resource_name:
+      RecordSet
```

Comment:
RecordSets are a little odd because we have to model them from `ChangeResourceRecordSets`:
- https://github.com/aws/aws-sdk-go/blob/main/models/apis/route53/2013-04-01/api-2.json#L1299

This operation takes in batches of multiple record sets at the same time and various actions (e.g. CREATE, DELETE, UPSERT) to specify what to do with these record sets. I decided to model the RecordSet resource as an individual CR instead of having a single CR representing multiple record sets to make it simpler for reconciliation. 
-